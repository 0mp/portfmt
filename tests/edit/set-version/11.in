DISTVERSION=	2.3.0
DISTVERSIONSUFFIX=	-Matrix
