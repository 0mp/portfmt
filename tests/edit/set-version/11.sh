${PORTEDIT} set-version 2.4.0-Matrix 11.in | diff -L 11.expected -L 11.actual -u 11.expected -
