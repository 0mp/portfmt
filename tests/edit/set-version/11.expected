DISTVERSION=	2.4.0
DISTVERSIONSUFFIX=	-Matrix
